ubuntu24.04/Dockerfile

```diff
@@ -30,9 +28,8 @@ ENV PATH /usr/local/go/bin:$PATH
 WORKDIR /work
 
 RUN git clone https://github.com/NVIDIA/gpu-driver-container driver && \
-    cd driver/vgpu/src && \
-    go build -o vgpu-util && \
-    mv vgpu-util /work
+    go build -C driver/vgpu/src -o vgpu-util && \
+    mv driver/vgpu/src/vgpu-util /work
```

Comment:
Forgot to copy the binary to the final image.

Reply:
Here , I am not building gpu-driver-util since we don't need in Ubuntu 24.04 vanilla driver containers

Reply:
Actually, this change is cosmetic. I'll revert it and keep the diff as minimal as possible